_toc.yml

```diff
@@ -0,0 +1,16 @@
+file: README
+sections:
+- file: governance
+- file: conduct/code_of_conduct
+  sections:
+  - file: conduct/enforcement
+  - file: conduct/faq
+  - file: conduct/reporting_events
+  - file: conduct/reporting_online
+  - file: CodeofConductJupyterDay
+  - file: CodeofConductJupyterDayOrganizer
+- file: people
+- file: newsubprojects
+- file: papers
+- file: trademarks
+- file: projectlicense
```

Comment:
The project license has header "License Terms", which implies that it is the license for the governance docs. Can you change the title (either just here, or even better in the actual doc itself) to something that indicates it is a license template for Jupyter subprojects? Perhaps "Project Jupyter License Template" or something?

Reply:
Up to now, I think it's been mildly confusing, but the filename has clarified it slightly. However, now the filename "projectlicense" is hidden from the user, so I think we should have a more clear title in the document itself.

Reply:
Yeah I agree - thus far I haven't tried to change any "content" like titles etc, but I agree with you they could be structured better. I will try to take a very lightweight pass at these so they are easier to follow

Reply:
I just tried updating the titles so that they're a bit more explicit, and also added an `intro.md` section that is now built in the docs (and we retain the `README.md` file without changes so the links still work if people are directly browsing the repo)